NotebookExamples/csharp/Samples/DataFrame-Getting Started.ipynb

```diff
@@ -0,0 +1,387 @@
+    "df[\"Ints\"] = (ints / 5) * 100;\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table><thead><th><i>index</i></th><th>DateTimes</th><th>Ints</th><th>Strings</th></thead><tbody><tr><td>0</td><td>2019-01-01 00:00:00Z</td><td>300</td><td>Bar</td></tr><tr><td>1</td><td>2019-01-01 00:00:00Z</td><td>2100</td><td>Foo!</td></tr><tr><td>2</td><td>2019-01-01 00:00:00Z</td><td>-1</td><td>Bar</td></tr></tbody></table>"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "// Fill nulls in our columns, if any. Ints[2], Strings[0] and Strings[1] are null\n",
+    "df[\"Ints\"].FillNulls(-1, inPlace: true);\n",
+    "df[\"Strings\"].FillNulls(\"Bar\", inPlace: true);\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table><thead><tr><th><i>index</i></th><th></th></tr></thead><tbody><tr><td>0</td><td>2019-01-01 00:00:00Z</td></tr><tr><td>1</td><td>300</td></tr><tr><td>2</td><td>Bar</td></tr></tbody></table>"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "// To inspect the first row\n",
+    "IList<object> row0 = df[0];\n",
+    "row0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table><thead><th><i>index</i></th><th>DateTimes</th><th>Ints</th><th>Strings</th></thead><tbody><tr><td>0</td><td>2019-01-01 00:00:00Z</td><td>300</td><td>Bar</td></tr><tr><td>1</td><td>2019-01-01 00:00:00Z</td><td>-1</td><td>Bar</td></tr></tbody></table>"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "// Filter rows based on equality\n",
+    "DataFrame filtered = df.Filter(strings.ElementwiseNotEquals(\"Foo!\"));\n",
+    "filtered"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table><thead><th><i>index</i></th><th>DateTimes</th><th>Ints</th><th>Strings</th></thead><tbody><tr><td>0</td><td>2019-01-01 00:00:00Z</td><td>-1</td><td>Bar</td></tr><tr><td>1</td><td>2019-01-01 00:00:00Z</td><td>300</td><td>Bar</td></tr><tr><td>2</td><td>2019-01-01 00:00:00Z</td><td>2100</td><td>Foo!</td></tr></tbody></table>"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "// Sort our dataframe using the Ints column\n",
+    "DataFrame sorted = df.Sort(\"Ints\", ascending: true);\n",
```

Comment:
Maybe we should rename `Sort` to `OrderBy` to match the `System.Linq` method.

Reply:
Right now? We could put up a 0.2.0

Reply:
No, not before the blog is out. Just something for the library's backlog. I've opened https://github.com/dotnet/corefxlab/issues/2788